modules/vcf/templates/annotate.sh

```diff
@@ -278,6 +278,80 @@ viab(){
 
 }
 
+fix_vep_str () {
+  #read the vcf round1: file to get the index of ALLELE_NUM in the CSQ annotation
+  while IFS= read -r line; do
+      # || echo "" to prevent exit upon non matching lines
+      csqDesc=$(expr match "$line" '##INFO=<ID=CSQ,Number=.,Type=String,Description="Consequence annotations from Ensembl VEP. Format: \(.*\)">') || echo ""
+      if [[ -n "$csqDesc" ]]; then
+        IFS='|' read -ra arr <<< "$csqDesc"
+        for index in "${!arr[@]}"; do
+          if [[ "${arr[index]}" == "ALLELE_NUM" ]]; then
+            alleleNumIdx="$index"
+          fi
+        done 
+        #exit the loop if CSQ was found
+        break
+      fi
+  done < <(zcat "vep_!{vcfOut}")
+
+  if [ -z "$alleleNumIdx" ]; then
+    >&2 echo -e "error: VCF is missing CSQ/ALLELE_NUM"
+    exit 1
+  fi
+
+  #read the vcf round2: fix annotations for lines containing multiple <CNV:TR>
+  while IFS= read -r line; do
+    #Skip headers
+    if [[ $line != \#* ]]; then
+      lineArray=(${line})
+      alts="${lineArray[4]}"
+      IFS=',' read -r -a altArray <<< "$alts"
+      #check if line contains CNV:TR ALT and multiple ALT values, else skip it
+      if [ "${altArray[0]}" = "<CNV:TR>" ] && [ ${#altArray[@]} -gt 1 ]; then
+        #combined <CNV:TR> and none <CNV:TR> lines should not exist, exit if encountered anyway.
+        for alt in "${altArray[@]}"; 
+        do 
+          if [[ "$alt" != "<CNV:TR>" ]]; then 
+          >&2 echo -e "error: VCF line contains mixed STR/nonSTR ALT alleles."
```

Comment:
if this happens this would be valid but unexpected vcf and then we should extend this workaround